Fix bullseye (#8)
YunoHost-Apps/pepettes_ynh: doc/DISCLAIMER.md

> * Other infos that people should be aware of, such as:
>     * How to configure this app: During the installation, or in `settings.py` after installation.

Reviewer: ```suggestion
This app may be configured through the `settings.py` after installation.
```
